Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -171,6 +169,15 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
             guard let self = self else { return }
             (self.view as? NSScrollView)?.contentView.contentInsets.bottom = self.bottomContentInsets
         }
+
+        NotificationCenter.default.addObserver(
+            forName: STTextView.didChangeSelectionNotification,
+            object: nil, 
+            queue: .main
+        ) { [weak self] _ in
+            guard let textLayoutManager = self?.textView.textLayoutManager else { return }
```

Comment:
This guard statement should be in the method itself so we can see quickly that it will fail if there is no layout manager.